test/runnable/sdtor.d

```diff
@@ -4742,6 +4742,85 @@ void test14078()
 
 /**********************************/
 
+void test67()
+{
+    char[] deleted;
+
+    struct S
+    {
+        char* p;
+
+        ~this() { deleted ~= *p; }
+
+        void opAssign(S rhs)
+        {
+            // swap
+            char* tmp = p;
+            this.p = rhs.p;
+            rhs.p = tmp;
+        }
+    }
+
+    char a = 'a', b = 'b';
+    {
+        S s = S(&a);
+        s = S(&b);
+    }
+    assert(deleted == "ab", deleted);
+}
+
+/**********************************/
+
+void test68()
+{
+    static struct S
+    {
+        int i;
+        bool opEquals(S) { return false; }
+        ~this() {}
+    }
```

Comment:
What's the point of this test ?

Reply:
I don't know. Copied from https://github.com/dlang/dmd/pull/10593/files which this is a reboot of.

Reply:
I presume @SSoulaimane had a reason, and there's no harm in it staying.

Reply:
I think there should be a test of `i` for it to make sense. And possibly a copy ctor.